types/office-js/index.d.ts

```diff
@@ -28874,15 +29062,36 @@ declare namespace Excel {
          * @remarks
          * [Api set: ExcelApi 1.16]
          */
-        root = "Root"
+        root = "Root",
+        /**
+         * Represents a `DoubleCellValue`.
+         *
+         * @remarks
+         * [Api set: ExcelApi 1.19]
+         */
+        double = "Double",
+        /**
+         * Represents a `StringCellValue`.
+         *
+         * @remarks
+         * [Api set: ExcelApi 1.19]
+         */
+        string = "String",
+        /**
+         * Represents a `BooleanCellValue`.
+         *
+         * @remarks
+         * [Api set: ExcelApi 1.19]
+         */
+        boolean = "Boolean"
     }
     /**
      * Represents the value in a cell.
      *
      * @remarks
      * [Api set: ExcelApi 1.16]
      */
-    type ReferencedValue = ArrayCellValue | EntityCellValue | RootReferenceCellValue;
+    type ReferencedValue = ArrayCellValue | EntityCellValue | RootReferenceCellValue | DoubleCellValue | StringCellValue | BooleanCellValue;
```

Comment:
also here - is this change in the office-js-preview file?

Reply:
Yes